src/main/java/core/basesyntax/Animal.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Animal extends Zoo implements Moveable, Feedable, Placeable {

```

Comment:
is it logical that Animal extends Zoo? 
try to read deeper about inheritance, aggregation and composition